marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/default/lib.sjs

```diff
@@ -33,6 +33,19 @@ function getSourceContext(sourceContext) {
   return sourceContext;
 }
 
+function getPath(sourceContext, connector, propertyName) {
+  let path;
+  if (xdmp.castableAs("http://www.w3.org/2001/XMLSchema", "NCName", propertyName)) {
+    path = `${sourceContext}${connector}${propertyName}`;
+  } else {
+    if (connector.includes("*:")) {
+      connector = "";
```

Comment:
Should this replace just the "*:" with blank instead of the whole connector as blank? 

A first glance, it looks like it could lose "/", which may be significant.

Reply:
Yes, that right, @ryanjdew. It should only replace "*:"